spec/toy_robot_spec.rb

```diff
@@ -12,24 +12,32 @@
       expect(subject.y).to eq(0)
     end
 
-    specify 'that ToyRobot f is set to EAST' do
-      expect(subject.f).to eq(:east)
+    specify 'that ToyRobot facing is set to NORTH' do
```

Comment:
Hi @itinsley ,
Yes, I looked at the spec again and it looks like by default the robot is placed at 0,0 and facing NORTH. Should I set it back to facing EAST? Did I write the test correctly?